addon/templates/components/frost-link.hbs

```diff
@@ -1,18 +1,29 @@
 {{#if (eq priority 'primary')}}
-    <div class='content'>
-      {{yield}}
-        <div class='svg'>{{frost-icon icon='frost/new-tab'}}</div>
-    </div>
+  <div class='content text'>
+    {{yield}}
+      <div class='svg'>{{frost-icon icon='frost/new-tab'}}</div>
+  </div>
 {{/if}}
 
 {{#if (eq priority 'secondary')}}
-    <div class='content'>
+  <div class='content'>
+    <div class='text'>
       {{yield}}
     </div>
+  </div>
 {{/if}}
 
 {{#if (eq priority 'tertiary')}}
-    <div class='content'>
+  <div class='content text'>
+    {{yield}}
+  </div>
+{{/if}}
+
+{{#if (eq priority 'action')}}
+  <div class='content '>
+    <div class=' text'>
+      <div class='svg'>{{frost-icon icon=icon}}</div>
       {{yield}}
-    </div>
+      </div>
+  </div>
 {{/if}}
```

Comment:
Can we add a default {{yield}} to this in the case where nothing is provided (for customization before new designs are added to the base)?